extensions/binding/org.eclipse.smarthome.binding.wemo/src/main/java/org/eclipse/smarthome/binding/wemo/handler/WemoLightHandler.java

```diff
@@ -393,13 +393,13 @@ private synchronized void onSubscription() {
     }
 
     private synchronized void removeSubscription() {
-        logger.debug("Removing WeMo GENA subscription for '{}'", this);
         if (service.isRegistered(this)) {
-            String subscription = null;
+        logger.debug("Removing WeMo GENA subscription for '{}'", this);
+            String subscription = "bridge1";
```

Comment:
While you are extracting this "magic" string, I think you should make it a constant with capital letters on top of the class, so it is easier to find.

Reply:
Sure, will change it to a constant.